postgres/rdspostgres/rdspostgres.go

```diff
@@ -72,15 +74,17 @@ func init() {
 // OpenPostgresURL opens a new RDS database connection wrapped with OpenCensus instrumentation.
 func (uo *URLOpener) OpenPostgresURL(ctx context.Context, u *url.URL) (*sql.DB, error) {
 	cf := new(rds.CertFetcher)
-	vals := u.Query()
-	u.RawQuery = vals.Encode()
 
+	database := u.EscapedPath()
```

Comment:
It turns out `RawPath` was empty; IIUC, `EscapedPath()` uses `RawPath` if it's available, otherwise computes it from `Path`. Apparently the godoc is wrong when it says `RawPath` is always set.

Reply:
@clausti filed this as golang/go#29662, which has been fixed at tip.